File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PatientCheckInsCreatePatientCheckInsRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PatientCheckInsCreatePatientCheckInsRequestItem {
    @JsonProperty("CheckInDate")
    public String checkInDate;
    @JsonProperty("CheckInLocationId")
    public Integer checkInLocationId;
    @JsonProperty("PatientLicenseNumber")
    public String patientLicenseNumber;
    @JsonProperty("RegistrationExpiryDate")
    public String registrationExpiryDate;
    @JsonProperty("RegistrationStartDate")
    public String registrationStartDate;
}

File: sdks/thunkmetrc-go/wrapper/models/patient_check_ins_create_patient_check_ins_request_item.go
package models

type PatientCheckInsCreatePatientCheckInsRequestItem struct {
    CheckInDate string `json:"CheckInDate,omitempty"`
    CheckInLocationId int `json:"CheckInLocationId,omitempty"`
    PatientLicenseNumber string `json:"PatientLicenseNumber,omitempty"`
    RegistrationExpiryDate string `json:"RegistrationExpiryDate,omitempty"`
    RegistrationStartDate string `json:"RegistrationStartDate,omitempty"`
}
